lib/HTFeed/PackageType/EMMA/Volume.pm

```diff
@@ -22,6 +23,16 @@ sub move_sip {
 
 }
 
+sub clean_sip_success {
+  my $self = shift;
+
+  $self->SUPER::clean_sip_success();
+  my $s3 = HTFeed::Storage::S3->new(
+    bucket => get_config('emma','bucket'),
+    awscli => get_config('awscli')
+  );
+  $s3->rm('/',"--recursive");
```

Comment:
I think we only want to remove the actual ingested item on success, rather than everything in the bucket

Reply:
I'd like to use the Volume's `get_mets_filename` and `get_zip_filename` methods to determine the names of the two files to remove from the bucket, but doing so seems to break multiple tests. So, I'm doing something along the lines of EMMA/Volume `move_sip`. Another commit in the works.